tests/wicked/basic/sut/t14_ifcfg_test_1_1.pm

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: wicked
+# Summary: Generic script/ifcfg/test-* runner
+#          These test containing config parsing checks.
+#
+# Maintainer: Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use Mojo::Base 'wickedbase';
+use testapi;
+
+sub run {
+    my ($self, $ctx) = @_;
+    my $ifc1 = $ctx->iface;
+    my $ifc2 = $ctx->iface2;
+
+    return if $self->skip_by_wicked_version(">=0.6.76");
```

Comment:
One minor comment here, wicked version on older sle15 [or sle12] should will keep to same package versions as well, correct?

Reply:
As long as the version of SLE has maintenance support, they will receive the same updates as newer versions.